src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const errorMessage = () => {
```

Comment:
```suggestion
const showErrorMessage = () => {
```
Functions should be named according to these rules
![image](https://github.com/mate-academy/js_promise_basic_DOM/assets/96409563/0b35d2f1-c807-4162-a636-d0802aab23ac)